### PR TITLE
Speedup startup

### PR DESCRIPTION
**Pull request by mroderick**
**TL;DR**
Use [`cache-require-paths`](https://github.com/bahmutov/cache-require-paths) to make Buster.JS start faster, ~700ms faster on my machine.

---

Cache node's `require` lookups for considerable speedup.

#### To verify

Measure time to run tests with `time` in a buster dependent project

```bash
time buster-test
```

Checkout this branch and use [`npm link`](https://docs.npmjs.com/cli/link) to make a buster dependent project use this version of Buster.

Repeat the time measurement a couple of times (first succesful one writes the `.cache-require-paths.json`file).

Calculate any improvement over current Buster.

##### Results

On my machine with a small project running tests only in node

```
281 tests, 336 assertions, 1 runtime
```

     | Before   | After
-----|----------|---------
real | 0m1.969s | 0m1.254s
user | 0m1.356s | 0m1.144s
sys  | 0m0.189s | 0m0.090s

Speedup: `1969 - 1254 = 715ms`

(caveats: on  my machine, in my project, using `node@0.10`, today)

I think the difference is significant enough to add this dependency.

### CODE REVIEW

**Comment by dominykas** [comment]
So far I have two problems with this

* the cache file is not cleared automagically - if I hadn't suspected this is the culprit, I'd probably wouldn't have figured it out, but after yet-another-npm-install the dep tree reshuffled and things started failing
* people who check their `node_modules` into source control might not be happy with an extra file appearing and changing (which includes hardcoded paths, which will fail on other people's machines as per above point)
